components/contact.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div class="box">
```

Comment:
Wrong color for bottom contacts form. 

On the top contact form color should be — #F2F6FF
On the bottom contact form color should be — #DBE6FF

![image](https://github.com/user-attachments/assets/9e7b151a-46ad-418a-8560-a64f9ba47abd)